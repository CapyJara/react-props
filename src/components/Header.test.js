import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Dog component', () => {
  it('renders a Dog', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
