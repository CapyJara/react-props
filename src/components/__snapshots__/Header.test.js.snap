// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog component renders a Dog 1`] = `
<header
  style={
    Object {
      "display": "flex",
      "height": "2em",
      "justifyContent": "space-between",
      "padding": "1em",
    }
  }
>
  <img
    src="/path/image"
    style={
      Object {
        "height": "100%",
      }
    }
  />
  <h1
    style={
      Object {
        "margin": 0,
      }
    }
  >
    My Dogs
  </h1>
</header>
`;
